### PR TITLE
fix typo OK response in capital letters instead of 'ok'

### PR DESCRIPTION
**Pull request by juandspy**
# Description

Fix typo where some payloads includes an "OK" response in capital letters instead of "ok".

Fixes [#1200](https://github.com/RedHatInsights/insights-results-aggregator/issues/1200)

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update
- Refactor (refactoring code, removing useless files)

## Testing steps

Run `sh test.sh`.

## Checklist
* [x] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: tests/rest/multiple_reports.go [hunk 1]
```diff
@@ -127,8 +127,8 @@ func expectNumberOfReports(f *frisby.Frisby, response MultipleReportsResponse, e
 
 // expectOkStatus tests whether the response (JSON) contains status attribute set to 'ok'
 func expectOkStatus(f *frisby.Frisby, response MultipleReportsResponse) {
-	if response.Status == "ok" {
-		f.AddError(fmt.Sprintf("Expected error status, but got '%s' instead", response.Status))
+	if response.Status != "ok" {
+		f.AddError(fmt.Sprintf("Expected ok status, but got '%s' instead", response.Status))
```
Comment (joselsegura) [inline tests/rest/multiple_reports.go:131]: Nice catch!

**Review by joselsegura — APPROVED** [review]

**Comment by Bee-lee** [comment]
LGTM
btw you can use the prepared check in the Makefile, called `before_commit`, this runs all the checks and more, it's in all our Go projects

**Review by Bee-lee — APPROVED** [review]

File: tests/rest/multiple_reports.go (hunk 1 above)
Comment (tisnik) [reply tests/rest/multiple_reports.go:131]: hmm yes, that's better, as (in theory) the status can be anything.

File: server/reports.go [hunk 2]
```diff
@@ -128,7 +128,7 @@ func fillInGeneratedReports(clusterNames []types.ClusterName, reports map[types.
 	}
 
 	// it must be ok now
-	generatedReports.Status = "OK"
+	generatedReports.Status = "ok"
```
Comment (tisnik) [inline server/reports.go:131]: probably worth to introduce new constant there?

**Review by tisnik — APPROVED** [review]
LGTM

**Comment by juandspy** [comment]
> btw you can use the prepared check in the Makefile, called `before_commit`, this runs all the checks and more, it's in all our Go projects
Thanks @Bee-lee , updated the checklist :+1:

File: server/reports.go (hunk 2 above)
Comment (juandspy) [reply server/reports.go:131]: just for this occurence or for all the "ok" status in the code?

Comment (tisnik) [reply server/reports.go:131]: yes, and exported `const` is reachable in tests as well.

**Pull request merged**